tezos/context/src/lib.rs

```diff
@@ -60,13 +60,13 @@ pub type ContextValue = Vec<u8>;
 pub type TreeId = i32;
 
 /// Tree in String form needed for JSON RPCs
-pub type StringTreeMap = BTreeMap<String, StringTreeEntry>;
+pub type StringDirectoryMap = BTreeMap<String, StringTreeObject>;
```

Comment:
Is it this a StringTreeObject or StringDirectoryObject? (I don't even remember for what this code was)

Reply:
What I mean, is, either the left side or the right side is wrong, because one says Tree and the other Directory. I think in this case both should be Tree but I am no 100% sure.

Reply:
@sebastiencs nevermind, I think the way it is now is correct. It is "directory" because it is the mapping that represents a directory, but the children are Tree's